### PR TITLE
Use RobotManager constructor instead of RobotInit -> master

### PR DESCRIPTION
**Pull request by drorta**

### CODE REVIEW

File: src/main/java/frc/robot/RobotManager.java
Comment (YHerm) [inline src/main/java/frc/robot/RobotManager.java:1]: There are warnings in the file (in the variables)

**Review by YHerm — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotManager.java [hunk 1]
```diff
@@ -28,10 +28,9 @@ public class RobotManager extends LoggedRobot {
 
 	private Command autonomousCommand;
```
Comment (YHerm) [inline src/main/java/frc/robot/RobotManager.java:29]: Put under the final variables

**Review by YHerm — CHANGES_REQUESTED** [review]

**Review by Stormageddon37 — DISMISSED** [review]
Small PRs best PRs

**Review by NoamRosenberg08 — DISMISSED** [review]

**Review by GilStein1 — APPROVED** [review]

File: src/main/java/frc/robot/RobotManager.java [hunk 2]
```diff
@@ -26,12 +26,11 @@ public class RobotManager extends LoggedRobot {
...
 
-	private Command autonomousCommand;
+	private final Robot robot;
 
-	private Robot robot;
+	private Command autonomousCommand;
 
-	@Override
-	public void robotInit() {
+	public RobotManager() {
```
Comment (TheOnion313) [inline src/main/java/frc/robot/RobotManager.java:33]: Make sure the dependencies for WPILib are updated to support this change in the constructor

Comment (drorta) [reply src/main/java/frc/robot/RobotManager.java:33]: It is

**Review by Nitay4 — APPROVED** [review]

**Review by NoamRosenberg08 — APPROVED** [review]

**Review by tnrz135 — APPROVED** [review]

**Pull request merged**